tests/regression/roles/sonic_lldp_global/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+ansible_connection: httpapi
+module_name: lldp_global
+
+
+tests:
+  - name: test_case_01
+    description: Add Global LLDP configuration
+    state: merged
+    input:
+      enable: true
+      multiplier: 9
+      system_name: CR_sonic
+      hello_time: 18
+      mode: receive
+      system_description: Sonic_CR1_device
+      tlv_select:
+        system_capabilities: false
+
+  - name: test_case_02
+    description: Update Global LLDP configuration
+    state: merged
+    input:
+      multiplier: 10
+      system_name: CR1_sonic
+      hello_time: 200
+      tlv_select:
+        system_capabilities: true
+
+  - name: test_case_03
+    description: Delete specific global lldp configurations
+    state: deleted
+    input:
+      multiplier: 10
+      system_name: CR1_sonic
```

Comment:
Please include deletion of 'tlv_select' option as mentioned in the module example.